src/session-builder.ts

```diff
@@ -55,7 +55,7 @@ export class SessionBuilder {
         const address = this.remoteAddress.toString()
         const serialized = await this.storage.loadSession(address)
         let record: SessionRecord
-        if (serialized !== undefined) {
+        if (serialized !== undefined && serialized != null) {
```

Comment:
I'd prefer checking `serialized !== null` to avoid the implicit conversions. (Since `null == undefined` is `true` we could just say `serialized != undefined` here and it would eliminate `null`, but the `!==` makes things more clear IMO.)